client/src/components/mapview/ModalContent.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import Button from '../shared/button/button.jsx'
 
-export default ({ address, price, photo }) => (
+export default ({ address, price, photo }) => ( //TODO refactor + add CSS
   <div>
     <img src={photo}/>
```

Comment:
A small suggest here: Render <img> after <price>.
This way users/we can see spot address/price without scrolling down and display is consistent with spots wo/img. Suggest to add a test spot wo/img to spots DB to see which way you prefer.

Reply:
I think this issue is resolved with the CSS that I've added since. Let me know if not!